app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] There is an error here, this is supposed to be the `Comment` model having class `Comment` but you have `Like`. Please correct this error by replacing `class Like < ApplicationRecord` with `class Comment < ApplicationRecord`